### PR TITLE
Making Email OTP Configuration UIs Consistent With New SMS OTP Configuration UIs

### PR DESCRIPTION
**Pull request by nipunsampath**
- **Application Sign-in Methods Page:**
    - Added a dedicated "Add Email OTP as a second factor" card.
- **Email OTP Settings Page**
    - Changed the unit of `Email OTP expiry time` to minutes from seconds.
    - Shortened the hint text by removing redundant phrases.
    - Reduced the length of number pickers.
    - Added a unit for `Email OTP length` which gets updated according to the value from the `Use only numeric characters for OTP` checkbox.
    - Made error messages for `Email OTP length` include the correct unit according to the checkbox value.

### Related Issues

- https://github.com/wso2-enterprise/asgardeo-product/issues/11923

### Screenshots

- Application Sign-in Methods Page:

Screenshot 2022-08-25 at 11 19 38

- Email OTP Settings Page:

Screenshot 2022-08-25 at 11 17 36

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by thivi — APPROVED** [review]

**Review by vihanga-liyanage — APPROVED** [review]

File: apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx [hunk 1]
```diff
@@ -285,7 +286,15 @@ export const SignOnMethods: FunctionComponent<SignOnMethodsPropsInterface> = (
                 ...authenticationSequence,
                 ...cloneDeep(SecondFactorTOTPSequenceTemplate)
             });
-        } else if (loginFlow === LoginFlowTypes.FIDO_LOGIN) {
+        } else if(loginFlow === LoginFlowTypes.SECOND_FACTOR_EMAIL_OTP){
```
Comment (LakshiAthapaththu) [inline apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx:289]: Check formatting

File: apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx [hunk 2]
```diff
@@ -285,7 +286,15 @@ export const SignOnMethods: FunctionComponent<SignOnMethodsPropsInterface> = (
...
+            eventPublisher.publish("application-sign-in-method-click-add", {
+                type: "second-factor-email-otp"
+            });
+            setModeratedAuthenticationSequence({
+                ...authenticationSequence,
+                ...cloneDeep(SecondFactorEMAILOTPSequenceTemplate)
+            });
+        }else if (loginFlow === LoginFlowTypes.FIDO_LOGIN) {
```
Comment (LakshiAthapaththu) [inline apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx:297]: Check formatting

File: apps/console/src/features/identity-providers/components/forms/authenticators/email-otp-authenticator-form.tsx [hunk 3]
```diff
@@ -220,6 +240,17 @@ export const EmailOTPAuthenticatorForm: FunctionComponent<EmailOTPAuthenticatorF
 
                 const moderatedName: string = name.replace(/_/g, ".");
 
+                if (name === IdentityProviderManagementConstants.AUTHENTICATOR_INIT_VALUES_EMAIL_OTP_EXPIRY_TIME_KEY){
```
Comment (LakshiAthapaththu) [inline apps/console/src/features/identity-providers/components/forms/authenticators/email-otp-authenticator-form.tsx:243]: Check formatting

File: apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx (hunk 1 above)
Comment (nipunsampath) [reply apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx:289]: Fixed with 57492cfc2bc379354647133894224a4278beddb0

File: apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx (hunk 2 above)
Comment (nipunsampath) [reply apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx:297]: Fixed with 57492cfc2bc379354647133894224a4278beddb0

File: apps/console/src/features/identity-providers/components/forms/authenticators/email-otp-authenticator-form.tsx (hunk 3 above)
Comment (nipunsampath) [reply apps/console/src/features/identity-providers/components/forms/authenticators/email-otp-authenticator-form.tsx:243]: Fixed with 57492cfc2bc379354647133894224a4278beddb0

**Pull request merged**